doc/devel/index.rst

```diff
@@ -13,10 +13,62 @@ Contribute
       of this document as it will have the most up to date installation
       instructions, workflow process, and contributing guidelines.
 
-Thank you for your interest in helping to improve Matplotlib! There are various
-ways to contribute: optimizing and refactoring code, detailing unclear
-documentation and writing new examples, reporting and fixing bugs and requesting
-and implementing new features, helping the community...
+:octicon:`heart;1em;sd-text-info` Thank you for your interest in helping to improve
+Matplotlib! :octicon:`heart;1em;sd-text-info`
+
+As a user, you can give valuable feedback by reporting bugs and suggesting features.
+
+.. _submitting-a-bug-report:
+.. _request-a-new-feature:
+
+.. grid:: 1 1 2 2
+
+   .. grid-item-card::
+      :class-header: sd-fs-5
+
+      :octicon:`bug;1em;sd-text-info` **Submit a bug report**
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+      If you find a bug in the code or documentation, do not hesitate to submit a
+      ticket to the
+      `Issue Tracker <https://github.com/matplotlib/matplotlib/issues>`_.
+
+      We have preloaded the issue creation page with a Markdown form that you can
+      use to provide relevant context. Thank you for your help in keeping bug reports
+      complete, targeted and descriptive.
+
+      .. button-link:: https://github.com/matplotlib/matplotlib/issues/new/choose
+            :expand:
+            :color: primary
+
+            Report a bug
+
+   .. grid-item-card::
+      :class-header: sd-fs-5
+
+      :octicon:`light-bulb;1em;sd-text-info` **Request a new feature**
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+      Please post feature requests to the
+      `Issue Tracker <https://github.com/matplotlib/matplotlib/issues>`_.
+
+      The Matplotlib developers will give feedback on the feature proposal. Since
+      Matplotlib is an open source project with limited resources, we encourage
+      users to then also :ref:`participate in the implementation <contribute_code>`.
+
+      .. button-link:: https://github.com/matplotlib/matplotlib/issues/new/choose
+         :expand:
+         :color: primary
+
+         Request a feature
+
+
+We also welcome you to get more involved with the Matplotlib project.
+There are various ways to contribute:
+optimizing and refactoring code, detailing unclear documentation and writing new
+examples, reporting and fixing bugs and requesting and implementing new features,
+helping the community...
+
```

Comment:
I was thinking in the direction of separating simple feedback in form of issues and feature requests from direct involvement in the project, because I believe people have a quite clear idea whether they want to do the one or the other.

How about adding sections
- Bug reports and feature requests
- Contributing to Matplotlib

and putting the above text in the second section?

Reply:
IMHO the main issue here is that "contributing" is used in two ways.

- General contribution, which is every interaction with the project
  https://matplotlib.org/devdocs/devel/index.html
- Getting involved with the project in the sense of actively working with the code/documentation/community.
  https://matplotlib.org/devdocs/devel/contribute.html

logical: A *general contribution* is one of two types
- posting an issue or PR
- actively working with the code/documentation/community

The first is more like giving a feedback with no intention of longer relationship. The second involves getting to know the project structures and is thus far more complex. I would really like to distinguish the two types, because the relevant information for the two target groups is significantly different and by clearly distinguishing between them, we can provide exactly the required information for each.

Reply:
> IMHO the main issue here is that "contributing" is used in two ways.
> 
> * General contribution, which is every interaction with the project (including openening an issue or feature request)
>   https://matplotlib.org/devdocs/devel/index.html
> * Getting involved with the project in the sense of actively working with the code/documentation/community.
>   https://matplotlib.org/devdocs/devel/contribute.html
> 
> logical: A _general contribution_ is one of two types
> 
> * posting an issue or feature request
> * actively working with the code/documentation/community
> 
> The first is more like giving a feedback with no intention of longer relationship. The second involves getting to know the project structures and is thus far more complex. I would really like to distinguish the two types, because the relevant information for the two target groups is significantly different and by clearly distinguishing between them, we can provide exactly the required information for each.

I feel we are missing a good terminology for the cases here.

Reply:
I've removed the sentence. The bugs and features cards have a shadow and an icon in the heading. IMHO this makes them visually distinct from other cards on the page - in particular the shadow also detaches the cards from the otherwise flat content of the page. I'm a bit hesitant to play with colors too much.